packages/reassure-cli/src/types.ts

```diff
@@ -0,0 +1,8 @@
+export type DefaultOptions = {
+  /** All logs */
+  verbose?: boolean;
+  /** No logs */
+  silent?: boolean;
+  /** No Ascii art */
+  'no-ascii-art'?: boolean;
```

Comment:
IMO I would drop this option, and display ascii art only either only for CLI init OR only for interactive prompts if we go that option.